source/renderer/app/components/sidebar/Sidebar.js

```diff
@@ -62,20 +76,73 @@ export default class Sidebar extends Component<Props> {
       name: CATEGORIES_BY_NAME.WALLETS.name,
     }).route;
 
-    if (menus && activeSidebarCategory === walletsCategory) {
+    const hardwareWalletsCategory = find(categories, {
+      name: CATEGORIES_BY_NAME.HARDWARE_WALLETS.name,
+    }).route;
+
+    if (
+      menus &&
+      menus.nonHardwareWallets &&
+      menus.nonHardwareWallets.wallets &&
+      activeSidebarCategory === walletsCategory
+    ) {
       subMenu = (
         <SidebarWalletsMenu
-          wallets={menus.wallets.items}
+          wallets={
+            menus.nonHardwareWallets && menus.nonHardwareWallets.wallets
+              ? menus.nonHardwareWallets.wallets.items
+              : []
+          }
           onAddWallet={onAddWallet}
-          onWalletItemClick={menus.wallets.actions.onWalletItemClick}
-          isActiveWallet={id => id === menus.wallets.activeWalletId}
+          onWalletItemClick={
+            menus.nonHardwareWallets && menus.nonHardwareWallets.wallets
+              ? menus.nonHardwareWallets.wallets.actions.onWalletItemClick
+              : null
+          }
+          isActiveWallet={id =>
+            id ===
+            (menus.nonHardwareWallets && menus.nonHardwareWallets.wallets
+              ? menus.nonHardwareWallets.wallets.activeWalletId
+              : null)
+          }
```

Comment:
@DeeJayElly as I mentioned above related to SidebarManus structure. Once the structure is simplified even this will be much simpler and without additional checks

Reply:
Fixed